package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-ora2",
-  "version": "0.0.1",
+  "version": "0.2.0",
```

Comment:
I think it is odd to bump it here, since there is no change in user functionality.